packages/tpetra/core/src/kokkos_refactor/Tpetra_KokkosRefactor_Details_MultiVectorDistObjectKernels.hpp

```diff
@@ -231,6 +242,16 @@ outOfBounds (const IntegerType x, const IntegerType exclusiveUpperBound)
          range_type (0, idx.size ()),
          PackArraySingleColumnWithBoundsCheck (dst, src, idx, col),
          errorCount);
+
+      // FENCE REVIEW - CONFIRMED FAILURE
+      //   Testing: This code is exercised by unit tests.
+      //   GTX960:  Passed (confirmed) with CUDA_LAUNCH_BLOCKING=0 and fence removed.
+      //   White:   Failed with CUDA_LAUNCH_BLOCKING=0 and fence removed.
+          // TpetraCore_CrsMatrix_ReplaceDomainMapAndImporter_MPI_4
+          // TpetraCore_ImportExport2_UnitTests_MPI_4
+      //   Plan:    Need to determine downstream failure point. Fix and remove this fence.
+      //   Notes:
+
```

Comment:
I don't see the same failure on ride with this fence removed

Reply:
with and without `CUDA_LAUNCH_BLOCKING=0`

Reply:
@tjfulle Thanks. I'll verify these on white and post the exact config and setup I was using. Something got broken with white's libs since this morning so many be delayed a bit.

Reply:
I do get other failures with `CUDA_LAUNCH_BLOCKING=0`.  I think unrelated.  I'll bring them up in today's Tpetra meeting.

Reply:
This one I could not replicate the fails. I will update the branch and comments when I rebase it to develop. Thanks.

Reply:
Updated the comments to reflect this.